blocks/news-filter/news-filter.css

```diff
@@ -0,0 +1,100 @@
+.news-filter form select {
+    font-size: var(--body-font-size-xs);
+    font-weight: bold;
+    text-transform: uppercase;
+    padding: 0 50px 0 20px;
+    margin: 3% 0;
+    color: var(--dark-gray);
+    background-color: white;
+    height: 40px;
+    cursor: pointer;
+    border: 1px solid var(--dark-gray);
+    background-image: url("/icons/down.svg");
+    background-size: 20px;
+    background-repeat: no-repeat;
+    background-position: right 15px center;
+    appearance: none;
+}
+
+.news-cards .card {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    border-top: 1px solid var(--light-grey);
+    flex-direction: column-reverse;
+}
+
+.news-cards .card .card-left,
+.news-cards .card .card-right {
+    display: flex;
+    padding: 10px 0;
+    align-items: flex-start;
+}
+
+.news-cards .card .card-left {
+    width: 90%;
+    flex-direction: column;
+}
+
+.news-cards .card .card-right {
+    margin-left: 10px;
+}
+
+.news-cards .card .card-body {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+    width: 80%;
+    height: auto;
+
+    p>span::after {
+        content: "-";
+        padding: 0 5px;
+    }
+
+    h3 {
+        margin: 0;
+    }
+}
+
+.news-cards .card .card-thumbnail {
+    width: 25%;
+    text-align: center;
+
+    picture {
+        width: 100%;
+    }
+}
+
+.news-cards .card .news-date {
+    font-size: 16px;
+    font-weight: 600;
+    color: #717171;
```

Comment:
Can we use --dark-gray instead of this (very slightly lighter) specific gray? We have too many colors going on.

Reply:
fixed